src/Controls/samples/Controls.Sample.UITests/MauiProgram.cs

```diff
@@ -41,5 +42,21 @@ protected override void OnAppLinkRequestReceived(Uri uri)
 		{
 			base.OnAppLinkRequestReceived(uri);
 		}
+
+		protected override Window CreateWindow(IActivationState activationState)
+		{
+#if WINDOWS
```

Comment:
Won't we have the same problem on catalyst?

Reply:
Yes, good point. Updated.